tts/Player.def.lua

```diff
@@ -72,6 +72,7 @@ function PlayerInstance.getSelectedObjects() end
 ---@field Pink tts__Player
 ---@field Black tts__Player
 ---@field Grey tts__Player
+---@field [tts__PlayerColor] tts__Player
```

Comment:
_Technically_ if Luanalysis is working properly, this shouldn't be needed. However, I'm fine with its inclusion.

Actually, ideally you'd be able to remove all explicit fields and just have this line. However, it'd break auto-completion. I'm not sure how I feel about trying to resolve complex types for auto-completion though.

P.S. You don't need to do anything to resolve this. Just adding a note for record keeping/discussion.

Reply:
Than I'm going to suggest LA isn't working properly.  Pretty sure I needed it.